### PR TITLE
Stats show corrected total games

### LINKED ISSUE #2350: Number of games at the https://hanab.live/scores
The number of games changed only on the screen where a game is started, shouldn't be the same number at score and statistics page (https://hanab.live/scores/) ?
Maybe there we should have something like:
Date joined: March 17th, 2020
Games played (in non-speedruns): __no of completed not 2p games __
2p games:
Terminated:
Time spent playing (in non-speedruns): 103 days, 6 hours, and 52 minutes
Games played (in speedruns): 42
Time spent playing (in speedruns): 43 minutes
Total max scores: 1271 (13.8%)

### PR DESCRIPTION
**Pull request by DrAnax**
Closes #2350

### CODE REVIEW

File: server/src/views/scores.tmpl [hunk 1]
```diff
@@ -6,7 +6,7 @@
   </li>
   <li>
     <span class="stat-description">Games played (in non-speedruns):</span>
-    {{.NumGames}}
+    {{.NumGamesTotal}} ({{.NumGamesNormal}} normally terminated)
```
Comment (Zamiell) [inline server/src/views/scores.tmpl:9]: shouldn't this say "normally finished"?

File: server/src/views/stats.tmpl [hunk 2]
```diff
@@ -27,7 +27,7 @@
               <ul>
                 <li>
                   <span class="stat-description">Games played (in non-speedruns):</span>
-                  {{.NumGames}}
+                  {{.NumGamesTotal}} ({{.NumGamesNormal}} normally terminated)
```
Comment (Zamiell) [inline server/src/views/stats.tmpl:30]: and here

**Comment by DrAnax** [comment]
Workers not running anymore?

**Comment by Zamiell** [comment]
it should be <https://github.com/Hanabi-Live/hanabi-live/blob/main/.github/workflows/ci.yml#L3>

i dont know what's wrong

**Comment by Zamiell** [comment]
oh nevermind i see the problem

**Comment by Zamiell** [comment]
i dont know how to re-run the job

**Comment by Zamiell** [comment]
but i think it passes so ill just merge, thank you

**Pull request merged**